acceptance/workspace/jobs/create/test.toml

```diff
@@ -0,0 +1,7 @@
+[[Server]]
+Pattern = "POST /api/2.1/jobs/create"
+Response.Body = '''
```

Comment:
Why Response.Body instead of just Body?

Reply:
That's more explicit. The body could be both the request body and the response body in the context of a server.